File: GestionInterim/app/src/main/java/com/example/gestioninterim/FragmentTitleSubTitleSlogan.java
package com.example.gestioninterim;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

public class FragmentTitleSubTitleSlogan extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View rootView = inflater.inflate(R.layout.fragment_title_subtitle_slogan, container, false);
        return rootView;

    }
}

File: GestionInterim/app/src/main/java/com/example/gestioninterim/FragmentTitleSubTitleSlogan.kt
package com.example.gestioninterim

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment

class FragmentTitleSubTitleSlogan : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_title_subtitle_slogan, container, false)
    }
}